src/components/Forms/ReservationForm.tsx

```diff
@@ -1,7 +1,9 @@
 import { Box, Button, TextField } from "@mui/material";
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import useAxios from "../../hooks/useAxios";
+import ReservationConfirmModal from "../Modals/ReservationConfirmModal/ReservationConfirmModal";
 
-const ReservationForm = () => {
+const ReservationForm = ({ data }: { data: any }) => {
```

Comment:
Try to avoid `any` types as much as possible. In actual projects you should never even use `any`